### PR TITLE
Add three more python deps from hatch.

### PR DESCRIPTION
**Pull request by dlorenc**
Fixes:

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [X] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [X] REQUIRED - The version of the package is still receiving security updates

### CODE REVIEW

**Comment by rawlingsj** [comment]
update check failed because of a newer version of brotli - but that's been [merged](https://github.com/wolfi-dev/os/blob/8794c10d0ee221d8931bd8ad0305c9c629d348ed/brotli.yaml#L3) since so let's ignore the failed check.

**Review by rawlingsj — DISMISSED** [review]

File: py3-SecretStorage.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Generated from https://pypi.org/project/SecretStorage/
+package:
+  name: py3-SecretStorage
```
Comment (rawlingsj) [inline py3-SecretStorage.yaml:3]: Is the camel case here intentional?

Comment (dlorenc) [reply py3-SecretStorage.yaml:3]: Yeah it's the real package name.

**Review by rawlingsj — APPROVED** [review]

File: py3-SecretStorage.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 3993fc7162fe50852852913b90b203ba13727c38
+      repository: https://github.com/mitya57/secretstorage
+      tag: ${{package.version}}
+
+  - name: Python Build
+    runs: |
```
Comment (vaikas) [inline py3-SecretStorage.yaml:35]: Should this and all the others use:
python/build-wheel pipeline instead?

Comment (dlorenc) [reply py3-SecretStorage.yaml:35]: Oh fancy! I didn't know we had that one yet.

**Pull request merged**